<launch>
    <node pkg="ssafy_2" type="mgeo_pub.py" name="mgeo_pub"  />
    <node pkg="ssafy_2" type="gpsimu_parser.py" name="gpsimu_parser" />
    <node pkg="ssafy_2" type="global_path_pub.py" name="global_path_pub"  />
    <node pkg="ssafy_2" type="local_path_pub.py" name="local_path_pub" />    
    <node pkg="ssafy_2" type="tf_pub.py" name="tf"  />
    <node pkg="ssafy_3" type="traffic.py" name="traffic" />
    <node pkg="ssafy_3" type="lattice_planner.py" name="lattice_planner" output="screen" args="/Object_topic"/>
    <node pkg="ssafy_3" type="advanced_purepursuit2.py" name="advanced_purepursuit"  args="/lattice_path"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ssafy_3)/rviz/lane_detection_rviz.rviz" />
</launch>
<!-- kcity traffic light 시나리오 launch 파일 -->